packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -402,18 +403,18 @@ final searchAutoCompleteKey = GlobalKey(debugLabel: searchAutoCompleteKeyName);
 /// Parts of active editing for auto-complete.
 class EditingParts {
   EditingParts({
-    this.activeWord,
-    this.leftSide,
-    this.rightSide,
+    this.activeWord = '',
+    this.leftSide = '',
+    this.rightSide = '',
```

Comment:
instead of adding default values, make required

Reply:
done